### DISCUSSION TITLE
Getting Error while Updating Pimcore

### DISCUSSION THREAD

**Discussion by vinitchopra12** [pimcore/pimcore#10448] [Q&A / Support]
Hello, 

When I try to update the pimcore using below command, it give me an error while updating. And, thus the Pimcore instance got stop working then.

Command: 
`COMPOSER_MEMORY_LIMIT=-1 composer update`

![image](https://user-images.githubusercontent.com/14720025/135240230-abf77883-ef4e-450d-a1fa-3487a0debf95.png)

I have tried with clearing cache n all, but no success.

**Comment by ComputerMasta** [comment]
The composer consumes all the available memory with that command , you should use command that limit composer memory usage.

`COMPOSER_MEMORY_LIMIT=512M php composer.phar update`

If you are a beginner at pimcore and need more information check [here](https://factory.dev/pimcore-knowledge-base/how-to/pimcore-for-beginners).
